wbooks-api/app/controllers/api/v1/rents_controller.rb

```diff
@@ -14,6 +14,7 @@ def show
       def create
         rent = Rent.new(rent_params)
         if rent.save
+          RentCreationEmailWorker.perform_async(current_user.id, rent.id)
```

Comment:
There is no need to create a worker to send the email. Look this [https://gist.github.com/maxivak/690e6c353f65a86a4af9](gist)